web/login.html

```diff
@@ -58,9 +58,9 @@
 
 		// load main module
 		app.require('main', function () {
-			$('#login, #password').on('keypress', function () {
+			$('#login, #password').on('keypress', function (e) {
 				$('#message').html('&nbsp;'); 
-				if (event.keyCode == 13) $('#submit').click();
+				if (((typeof e.which == "number") ? e.which : e.keyCode) == 13) $('#submit').click();
```

Comment:
I tested in latest firefox (I am on OSX) and it seems to be working fine. If it is indeed not working on FireFox (win8), then there are lots of problems in other places, as I use it this way in fields and other places. Is it only problem of FireFox on win8 or other versions too. Also, is it possible that jQuery polifils it and then it works if events attached through jQuery.